src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlStyle.java

```diff
@@ -441,6 +441,12 @@
      */
     colSummaryItemName,
 
+    /**
+     * The class of the cells in a table column used to display additional
+     * information without any particular style.
+     */
+    colExtra,
```

Comment:
The `extra` in the same seems irrelevant. How about `colPlain`, to indicate it is a column with no additional style.

Reply:
I was considering `colPlain`, but dismissed it because I thought we or somebody else might actually add styles to it at some point. But I'll take the risk of future cognitive dissonance and go with `colPlain`.